### ISSUE TITLE
download information on who edit osm feature and when

### ISSUE THREAD

**Issue by michielvandijk** [ropensci/osmdata#154]
To validite/combine the osm features with other information I have downloaded with the osmdata package (great stuff!), it would be helpful if I could get information on when the features were added to osm or when they were edited for the last time (and perhaps also by whom). I do not think this is possible (yet) with the package. When I query features in osm this information is always shown automatically so it must be easily available. Could this be added?

**Comment by mpadge** [comment]
It can't be added to the package at present, because `osmdata` gets data from the `overpass` server, and that does not provide change logs, which are what contain that information. Changes logs are (still) only available via full or partial planet dump files, such as from `geofabrik.de`. There are mutterings of putting change log data in overpass, but I don't think there are any concrete plans let alone timetables. Sorry about that.

**Comment by michielvandijk** [comment]
Too bad. Alternatively, would it be possible to obtain the 'Way' ID of the polygons that are downloaded. I assume they must be available if they are downloaded. Theoretically, it should then be possible to go to the corresponding website (https://www.openstreetmap.org/way/,,,) and scrape the additional information from there.

**Comment by mpadge** [comment]
Yeah, that's easy. There's actually a new-ish function, courtesy of @Robinlovelace, called [`opq_osm_id()`](https://ropensci.github.io/osmdata/reference/opq_osm_id.html), detailed in [this issue](https://github.com/ropensci/osmdata/issues/95). What I could add to that would be an option, `openurl = FALSE` which, when `TRUE`, would convert the ID to an `openstretmap.org` url and open it in a browser. That would then enable immediate inspection of the change history. If you think that would help, then please re-open the issue and I'll implement that. @Robinlovelace do you think this would be a good enhancement?

**Comment by michielvandijk** [comment]
I just discovered that osm_id, which is part of the osm_polygon df is the way number. Did not realize this before. So in fact I could start scrapping the change history/latest entry. But having the possibility to click on a link that opens a browser for quick inspection would be a very nice feature.

**Comment by mpadge** [comment]
@michielvandijk The commit that closed #95 linked to above implements this:
```
opq_osm_id(id = c (136190595, 136190596), type = "way", open_url = TRUE)
```
... and the browser pages will open up. Hope that helps!
